### PR TITLE
Fixed #33994 -- Corrected position of extrastyle and extrahead blocks in admin base template.

### PR DESCRIPTION
**Pull request by j-the-ripper**

### CODE REVIEW

File: django/contrib/admin/templates/admin/base.html [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
 {% if LANGUAGE_BIDI %}<link rel="stylesheet" href="{% block stylesheet_rtl %}{% static "admin/css/rtl.css" %}{% endblock %}">{% endif %}
-{% block extrahead %}{% endblock %}
 {% block responsive %}
     <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0">
     <link rel="stylesheet" href="{% static "admin/css/responsive.css" %}">
     {% if LANGUAGE_BIDI %}<link rel="stylesheet" href="{% static "admin/css/responsive_rtl.css" %}">{% endif %}
 {% endblock %}
 {% block blockbots %}<meta name="robots" content="NONE,NOARCHIVE">{% endblock %}
+{% block extrahead %}{% endblock %}
+{% block extrastyle %}{% endblock %}
```
Comment (felixxm) [inline django/contrib/admin/templates/admin/base.html:23]: `extrahead` may contain elements that override extra styles, so we should keep them in the same order for backward compatibility:

```suggestion
{% block extrastyle %}{% endblock %}
{% block extrahead %}{% endblock %}
```

**Review by felixxm — COMMENTED** [review]
@j-the-ripper Thanks for this patch :+1:

**Comment by felixxm** [comment]
@j-the-ripper Thanks :+1: Welcome aboard :boat:

**Comment by j-the-ripper** [comment]
> @j-the-ripper Thanks +1 Welcome aboard boat

Thanks @felixxm It's my first PR and pardon for amateur PR description.

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Pull request merged**